go_modules

```diff
@@ -151,7 +151,7 @@ def cmd_go_mod(cmd, dir):
         error = e.output.decode("utf-8").strip()
         if error:
             log.info(error)
-        return False
+        raise
```

Comment:
well, then the return True on line 149 can be removed as well

I agree that would make it slightly more pythonic.

Reply:
@dirkmueller done